Content.Client/UserInterface/Systems/Chat/ChatUIController.cs

```diff
@@ -505,6 +507,14 @@ private void UpdateChannelPermissions()
             FilterableChannels |= ChatChannel.AdminAlert;
             FilterableChannels |= ChatChannel.AdminChat;
             CanSendChannels |= ChatSelectChannel.Admin;
+            FilterableChannels |= ChatChannel.CollectiveMind;
```

Comment:
move this one line up to not break the style

Reply:
We really shouldn't be hardcoding in new channels like this for particular gamemodes tbh, it's just going to become an even bigger mess.